### PR TITLE
docs(CardDAV): allow disabling of the system address book

### PR DESCRIPTION
**Pull request by miaulalala**
Documentation for https://github.com/nextcloud/server/pull/39925

### 🖼️ Screenshots
| Before | After |
|--------|--------|
| image | image
 |

### CODE REVIEW

**Review by hamza221 — APPROVED** [review]

**Review by st3iny — APPROVED** [review]
Looks good. Should we wait until the upstream feature was merged in case of changes?

**Comment by miaulalala** [comment]
Changed some more text, the config value has been renamed in the linked pr.

**Review by st3iny — APPROVED** [review]

**Pull request merged**
